developer_README.md

```diff
@@ -1,29 +1,195 @@
+- `LOCAL_APP_FILES_DIR`: base directory of your local App files (*auxiliary*)
+- `OUTPUT_FILE`: path to output file of your App
+- `APP_ARTIFACTS_DIR`: base directory for writing App artifacts
+
+You can adjust these environment variables by adjusting the file `./.env`.
+
+The file `./.env` is **hidden** by default in `RStudio`! You can show it by
+
+1. Tab `Files` in `RStudio`
+1. `More` Button in the Files-Toolbar
+1. Activate _Show Hidden Files_
+
+## MoveApps App Bundle
+
+Which files will be bundled into the final App running on MoveApps?
+
+- the file `./RFunction.R`
+- all directories defined in your `appspec.json` at `providedAppFiles` 
+
+Nothing else.
+
+## App development
+
+1. Execute `Rscript sdk.R` (on a terminal) or run/source `sdk.R` in _RStudio_
+1. Ensure the sdk executes the vanilla template App code. Everything is set up correctly if no error occurs and you see something like _Welcome to the MoveApps R SDK._
+1. Begin with your App development in `./RFunction.R`
+
+## Examples
+
+### Request App configuration from your users
+
+`./appspec.json`: define the settings UI on MoveApps. Users of your App can enter their configuration values.
 
+![img.png](documentation/app-configuration-ui.png)
+
+You can also use our [Settings Editor](https://www.moveapps.org/apps/settingseditor) to generate the App configuration
+
+```
+"settings": [
+ {
+   "id": "line_width",
+   "name": "Line width",
+   "description": "The width of the lines in the plot.",
+   "defaultValue": 2,
+   "type": "INTEGER"
+ },
+ {
+   "id": "legend",
+   "name": "Include legend?",
+   "description": "Should the plot contain a legend?",
+   "defaultValue": false,
+   "type": "CHECKBOX"
+ }
+],
+```
+
+`./app-configuration.json`: this is only needed during the app development to simulate an App run
+
+```
+{
+  "line_width": 2,
+  "legend": true
+}
+```
+
+`./RFunction.R`: your App will be called with the user's App configuration
+
+```
+rFunction = function(data, line_width, legend, ...) {
+}
+```
+
+`./tests/testthat/test_RFunction.R`: do not forget to test your App
+
+```
+test_data <- test_data("input3_stork.rds")
+
+test_that("happy path", {
+  actual <- rFunction(data = test_data, sdk = "unit test", year = 2005)
+  expect_equal(unique(lubridate::year(actual@timestamps)), 2005)
+})
+
+test_that("year not included", {
+  actual <- rFunction(data = test_data, sdk = "unit test", year = 2023)
+  expect_null(actual)
+})
+```
+
+### Produce an App artefact
+
+Your App can write files which the user can download after it has run.
+
+`./appspec.json`
+
+```
+  "createsArtifacts": true,
+```
+
+`./RFunction.R`
+
+```
+pdf(appArtifactPath("MorningReport_overviewTable.pdf"), paper = "a4r")
+```
+
+**Notice:** Only files are permitted to act as MoveApps App artifact! If your app produces a directory as an App artificat you have to bundle it eg. by zipping it. In other words: at the moment your App completes its work there must be only files present in `APP_ARTIFACTS_DIR`.
```

Comment:
@andreakoelzsch : sobald ihr in der anderen App einen sauberen Weg gefunden habt mehrere Dateien als ZIP-Artifact zu erzeugen könnt ihr bitte dieses Snippet hier in die Doku einfügen?

Reply:
done